components/Card.js

```diff
@@ -0,0 +1,21 @@
+import Itemlist from "./Itemlist";
+import styled from "styled-components";
+
+export default function Card() {
+  return (
+    <>
+      <StyledCard>
```

Comment:
The naming is irritating. 
I am expecting a Card here (single, one shoe). 
But I do get a Container for a lot of items  (please use a List HTML element for that)